test/sampledata/dataspace/MasterDataspace/study/datasets/dataset5004.tsv

```diff
@@ -1 +1 @@
-SubjectId	SequenceNum	date	QCStateLabel	visit_day	assay_identifier	summary_level	specimen_type	antigen antigen_type    virus   virus_insert_name   virus_type	clade	neutralization_tier tier_clade_virus	target_cell	initial_dilution	titer_ic50	titer_ic80	response_call	nab_lab_source_key	lab_code	exp_assayid
+SubjectId	SequenceNum	date	QCStateLabel	visit_day	assay_identifier	summary_level	specimen_type	antigen antigen_type    virus   virus_insert_name   virus_type	clade	neutralization_tier tier_clade_virus	target_cell	initial_dilution	titer_ic50	titer_ic80	titer_ID50	titer_ID80	nab_response_ID50	nab_response_ID80	response_call	nab_lab_source_key	lab_code	exp_assayid	slope
```

Comment:
Cory pointed out during estimation that the dataset000* files may not be needed.  If indeed so, it will be easier to remove all of them instead of having to maintain them.